CLI_QUICKSTART.markdown

```diff
@@ -344,4 +344,18 @@ with a pkill command:
 $ pkill vc-server
 $ pkill vc-pas
 ```
+##  IP addresses and ports used by various Veracruz components
+
+|    IP      | Port |                              Usage                                                         | Location(s) |
+|------------|------|--------------------------------------------------------------------------------------------|-------------|
+| 127.0.0.1  | 6000 | Port used by Veracruz Server and Runtime Manager to communicate over a TCP socket          | `veracruz-server/src/veracruz_server_linux.rs` |
```

Comment:
Can you put backticks around all of the IP addresses, here, and throughout the rest of the file?